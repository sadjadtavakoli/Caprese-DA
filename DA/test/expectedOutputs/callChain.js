
#2 function firstOne-test/unit_tests/callChain.js-2-5 entered from arrowFunction-test/unit_tests/callChain.js-1-17
#8 function secondOne-test/unit_tests/callChain.js-8-11 entered from firstOne-test/unit_tests/callChain.js-2-5
#13 function fifthOne-test/unit_tests/callChain.js-13-15 entered from secondOne-test/unit_tests/callChain.js-8-11
#13 function fifthOne-test/unit_tests/callChain.js-13-15 exited to function secondOne-test/unit_tests/callChain.js-8-11
#8 function secondOne-test/unit_tests/callChain.js-8-11 exited to function firstOne-test/unit_tests/callChain.js-2-5
#2 function firstOne-test/unit_tests/callChain.js-2-5 exited to function arrowFunction-test/unit_tests/callChain.js-1-17
#end Execution