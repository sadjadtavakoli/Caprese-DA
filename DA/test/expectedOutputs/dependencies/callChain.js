{"firstOne-test/unit_tests/callchain.js-2-5":{"callers":["t_1"],"tests":[]},"secondOne-test/unit_tests/callchain.js-8-11":{"callers":["t_22"],"tests":[]},"fifthOne-test/unit_tests/callchain.js-13-15":{"callers":["t_50"],"tests":[]},"keyMap":{"t_1":"callChain.js-test/unit_tests/callchain.js-1-17","t_22":"firstOne-test/unit_tests/callchain.js-2-5","t_50":"secondOne-test/unit_tests/callchain.js-8-11","t_53":"fifthOne-test/unit_tests/callchain.js-13-15"}}