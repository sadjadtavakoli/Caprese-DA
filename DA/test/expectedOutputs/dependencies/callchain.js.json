{"firstOne-test/unit_tests/callchain.js-2-5":{"callers":[],"tests":["t_1"]},"secondOne-test/unit_tests/callchain.js-8-11":{"callers":[],"tests":["t_2"]},"fifthOne-test/unit_tests/callchain.js-13-15":{"callers":[],"tests":["t_5"]},"keyMap":{"t_1":"callchain.js-test/unit_tests/callchain.js-1-17","t_2":"firstOne-test/unit_tests/callchain.js-2-5","t_5":"secondOne-test/unit_tests/callchain.js-8-11","t_8":"fifthOne-test/unit_tests/callchain.js-13-15"}}